css/header.css

```
@@ -0,0 +1,104 @@
+header {
+  background: url(/images/header_background.png);
+}
+
+header .container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+header .header__logo {
+  padding: 16px 10px;
+}
+
+header .header__menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+header a {
+  color: var(--darkgray);
+}
+
+.header__menu > ul {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  column-gap: 10px;
+}
+
+.header__menu > ul > li {
+  position: relative;
+}
+
+.header__menu > ul > li > a {
+  padding: 12px 18px;    
+}
+
+.header__menu > ul > li:hover > a {
+  background-color: var(--gray);
+}
+
+.header__menu > ul > li:last-child > a {
+  background-color: var(--darkorange);
+  color: var(--white);
+  padding: 12px 16px;
+}
+
+.header__menu > ul > li:last-child:hover a {
+  background-color: var(--gray);
+  color: var(--darkgray)
+}
+
+.header__menu > ul > li > a:after {
+  position: absolute;
+  content: '+';
+  margin-left: 5px;
+}
+
+.header__menu > ul > li a:only-child::after {
```

`+` indication should be removed in Shopping Cart button as well
```suggestion
.header__menu > ul > li a:only-child::after,
.header__menu > ul > li:last-child a::after {
```